util/qartod_service.py

```diff
@@ -0,0 +1,123 @@
+import json
+import engine
+import logging
+import requests
+from copy import deepcopy
+
+log = logging.getLogger(__name__)
+
+
+class QartodTestServiceAPI(object):
+    def __init__(self, qartod_url):
+        self.base_url = qartod_url
+
```

Comment:
Only difference I see between the definitions of find_qartod_tests and find_qartod_tests_bulk is how you populate "url" with either 'find' or 'bulk'. Is there a way you could make 1 method that is flexible enough to pick the appropriate value to append th self.base_url?

Reply:
I noticed another difference that may prevent that suggestion: the passing and usage of parameter or parameters. So disregard as needed.